staging/src/k8s.io/apiserver/pkg/admission/metrics/testutil_test.go

```diff
@@ -24,8 +24,15 @@ import (
 )
 
 func labelsMatch(metric *ptype.Metric, labelFilter map[string]string) bool {
-	for _, lp := range metric.GetLabel() {
-		if value, ok := labelFilter[lp.GetName()]; ok && lp.GetValue() != value {
+	for labelName, labelValue := range labelFilter {
```

Comment:
You may change the logic of labelsMatch, is the original approach intentional, and is there a lot of methods use `labelsMatch`?

Reply:
I'm not sure about the intention of the first approach, I added the person who wrote the code as cc in the issue. Just some [test methods](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics_test.go) implicitly use this method.

Reply:
/cc @jpbetz, does this change seem okay to you?